src/v3/src/components/List/List.tsx

```diff
@@ -92,15 +92,24 @@ const List: UISchemaElementComponent<{
       marginBlockEnd={4}
     >
       { options.description && <Typography component="p">{options.description}</Typography> }
-      <ListOdyssey listType={options.type ?? 'unordered'}>
+      <ListMui
+        component={options.type}
+        sx={{ listStyleType: 'decimal', pl: 4 }}
```

Comment:
is this style the same as the old odyssey version?

Reply:
spacing-wise it looks the same but I'm glad you commented because I did not properly handle the list style for the `ul` and `dl` types!